.gitignore

```diff
@@ -1,9 +1,13 @@
+*~
 *.pyc
+*.pyo
 *.egg-info
 *.swp
-.coverage
-*~
 build
 dist
-htmlcov
 .tox
+.eggs
+man/
```

Comment:
What generates `./man/` and why?

Reply:
good question, I guess this is just a C&P typo .. I will remove it.